### PR TITLE
Hotfix/users names

### PR DESCRIPTION
**Pull request by LiorAgnin**
## Proposed changes

subj

## Types of changes

What types of changes does your code introduce?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Refactor or a modification (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: dapp/src/sagas/communityEntities.js [hunk 1]
```diff
@@ -269,23 +272,6 @@ export function * fetchFeaturedCommunityEntitiesCount ({ communityAddress }) {
   }
 }
 
-function * fetchUserMetadata ({ account }) {
```
Comment (leonprou) [inline dapp/src/sagas/communityEntities.js:272]: So we need this or node?

**Review by leonprou — COMMENTED** [review]
This both for old and new communities, right?

File: dapp/src/sagas/communityEntities.js (hunk 1 above)
Comment (LiorAgnin) [reply dapp/src/sagas/communityEntities.js:272]: No, It used to fetch the user profile from 3box.

Comment (leonprou) [reply dapp/src/sagas/communityEntities.js:272]: So can you delete the whole function? Of you prefer to keep the changes minimal to just comment the body?

Comment (LiorAgnin) [reply dapp/src/sagas/communityEntities.js:272]: I've deleted the function.

**Review by leonprou — APPROVED** [review]
👍

**Pull request merged**